cvat-core/src/annotations-actions.ts

```diff
@@ -28,12 +29,20 @@ export enum ActionParameterType {
     NUMBER = 'number',
 }
 
+// For SELECT values should be a list of possible options
+// For NUMBER values should be a list with [min, max, step],
+// or a callback ({ job }) => [min, max, step]
 type ActionParameters = Record<string, {
     type: ActionParameterType;
-    values: string[];
+    values: string[] | (({ job }: { job: Job }) => string[]);
```

Comment:
What if we run action on a task? 
It is not possible on UI, but actions API assumes such ability. What job will be passed? 

It is a reason, why better to pass `instance: Job | Task`.